### PR TITLE
New package: gtk-doc

### PR DESCRIPTION
**Pull request by qwertos**

### CODE REVIEW

File: var/spack/repos/builtin/packages/gtk-doc/package.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+class GtkDoc(AutotoolsPackage):
+    """GtkDoc is a tool used to extract API documentation from
+    C-code like Doxygen, but handles documentation of GObject
+    (including signals and properties) that makes it very
+    suitable for GTK+ apps and libraries. It uses docbook for
+    intermediate files and can produce html by default and
+    pdf/man-pages with some extra work."""
+
+    homepage = "https://wiki.gnome.org/DocumentationProject/GtkDoc"
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/gtk-doc/package.py]: I would still have a `url` up here, as it is used by `spack versions` and `spack checksum`

File: var/spack/repos/builtin/packages/gtk-doc/package.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    # variant('pdf', default=False, description='Adds PDF support')
+
+    depends_on('autoconf', type='build')
+    depends_on('automake', type='build')
+    depends_on('libtool',  type='build')
+    depends_on('m4',       type='build')
+    depends_on('pkgconfig', type='build')
+
+    depends_on('python@3.2:')
+    depends_on('py-pygments')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/gtk-doc/package.py]: These two are probably both type=build/run

File: var/spack/repos/builtin/packages/gtk-doc/package.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    depends_on('libxslt')
+    depends_on('libxml2')
+    depends_on('docbook-xsl')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/gtk-doc/package.py]: What deptype should these be? Are they used at run-time? Or do they just link to the libraries?

File: var/spack/repos/builtin/packages/gtk-doc/package.py (hunk 1 above)
Comment (qwertos) [reply var/spack/repos/builtin/packages/gtk-doc/package.py]: URL re-added

File: var/spack/repos/builtin/packages/gtk-doc/package.py (hunk 2 above)
Comment (qwertos) [reply var/spack/repos/builtin/packages/gtk-doc/package.py]: Fixed

**Comment by alalazo** [comment]
@qwertos Can you please try to rebase? Travis is a bit of a pain these days...

**Comment by adamjstewart** [comment]
Travis seems to be passing, force-merging...

**Pull request merged**